DOC-425 part 6: Pindel, Pindel Annotation, SeSAMe Methylation Beta Estimation, Seurat - 10x Chromium, STAR - Counts, STAR 2-Pass Chimeric, STAR 2-Pass Genome, STAR 2-Pass Transcriptome, STAR-Fusion (#364)
NCI-GDC/gdc-docs: docs/Encyclopedia/pages/STAR_Counts.md

# STAR - Counts

## Description ##

STAR - Counts is an RNA expression pipeline used in GDC RNA-Seq harmonization.

## Overview ##

> STAR - Counts is a pipeline used for to quantify RNA gene and exon expression from unharmonized or GDC harmonized data. Quantification is performed with STAR - Counts using tumor and normal alignments and generates transcriptome profiling data.

Reviewer: ```suggestion
STAR - Counts is a pipeline used for to quantify gene expression from RNA-Seq data. Quantification is performed with STAR using tumor or normal alignments and generates transcriptome profiling data.
```

Author: changed

### Input

> * Tumor BAM - RNA-Seq

Reviewer: ```suggestion
* Genomic BAM - RNA-Seq
```

Author: changed

> * Normal BAM - RNA-Seq

Reviewer: ```suggestion
```
